### PR TITLE
1 default colors

### PR DESCRIPTION
**Pull request by jhogue**
First draft to solve Issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+All UI team members are expected to help maintain this scaffold and update best practices as we work on other projects. The workflow should be something like this: 
+
+* A team member clones this repo and starts a new project with it
+* They notice something that they need to change in the scaffold that is NOT specific for this project, like a default markup pattern has changed, a browser has become unsupported, or an error or ommission in a function needs an update
+* The team member changes it in their new project, BUT ALSO goes back to the scaffold and Files an Issue in Github and assigns it to themselves. 
+* Then can then resolve the ticket by making the change needed and submitting a PR for review, or, they can go back and handle the Issue later. 
+
+In this way, maintenance becomes a team effort and no one person is burdened with all maintenance responsibilities. 
+
+Likewise, if there is a new great idea that we should incorporate into this scaffold, any team member can file and Issue and bring the idea forward for review and inclusion.
```
Comment (nicholemaureen) [inline README.md]: file an\* issue

**Comment by nicholemaureen** [comment]
Nice!

File: base/_colors.scss [hunk 2]
```diff
@@ -1,41 +1,66 @@
...
+	// Brand should contain this project's specific color scheme
+	brand: (
+		base: #d62901, // Oomph red
+		dark-red: #a31f02
+	),
+	
+	// Default color scheme, edit values in relation to the numeric representation
+	// In other words, keep high values dark and low values light
+	mono: (
+		black: #131313,
```
Comment (johncionci) [inline base/_colors.scss:32]: Using keywords will throw an error via our scss-lint.

File: base/_colors.scss [hunk 3]
```diff
@@ -1,41 +1,66 @@
...
+		90: #303030,
+		75: #4f4f4f,
+		66: #666,
+		base: #888,
+		33: #999,
+		25: #ccc,
+		10: #f2f2f2,
+		white: #fff
```
Comment (johncionci) [inline base/_colors.scss:40]: Same here :)

File: base/_colors.scss (hunk 2 above)
Comment (jhogue) [inline base/_colors.scss:32]: Huh? So, what is allowed, or, better yet, how can we turn that off?

Comment (johncionci) [inline base/_colors.scss:32]: Currently, all the linters that we have active throw warnings, we could up that to certain ones throwing errors via 'severity notices'

So theres two that have to do with that warning:
https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword
https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorvariable

I guess we can let the masses decide? or maybe you can give scss-lint.yml a preview and let me know if you have any ideas.

Side note - I always understand black to be #000

Comment (jhogue) [inline base/_colors.scss:32]: Your side note is important… While the design may never use full black
#000, all you should have to worry about is that you want to use the color
defined as black. I think that is the magic of abstraction… same thing with
white. For whatever reason, if you want to define the whitest/lightest
color to be #f2f2f2, you can do that. The dev then only has to concern them
selves with the intent of the color, you know?

Comment (jhogue) [inline base/_colors.scss:32]: Per the Linter, I think the Linter will not throw an error here. What they
are trying to find is the use of HTML named colors, like `color: green`.
This should not preclude us from defining and using `color: palette(brand,
green)`

**Comment by johncionci** [comment]
Thoughts on this syntax?, it worked pretty decent for UWRI - it effectively limits the color palette so we don't get out of control.

```
reds: (
  lighter: #FF3333 // 20%
  light: #FF1A1A, // 10%
  base: #FF0000,
  dark: #E60000, // 10%
  darker: #CC0000, // 20%
)
```

Effectively the most we ever have of any given color is 5 tones/shades.
Pretty useful tool I find to get the values of lightened/darkened colors http://www.hexcolortool.com/#ff0000

Also gives a quick reference to the color we are working with "reds" for people that cant read hex, like me :)

**Comment by jhogue** [comment]
I think that is nice for a case where we need shades of the same color.
Pretty often, though, we don't, or can more easily use a `lighten()` or
`darken()` function where we need to tweak a color. I'm ok with adding that
in there as an example that someone can use if they need it, though.

**Comment by johncionci** [comment]
@jhogue before we push this through, can you run this through the latest linter and maybe open that pandoras box of what we should/shouldnt lint for?

**Comment by johncionci** [comment]
@jhogue sorry, still refs to old colors in here...
ex. typography.scss `$table-border-color: palette(mono, dove-gray);`

The linter wouldnt catch that, but Brunch would, but it would be in a project already...

**Pull request merged**